app/src/components/map/MapContainer.tsx

```diff
@@ -1,31 +1,48 @@
 import React from 'react';
 import 'leaflet/dist/leaflet.css';
 import 'leaflet-draw/dist/leaflet.draw.css';
-import { MapContainer as LeafletMapContainer, TileLayer, LayersControl, FeatureGroup, GeoJSON } from 'react-leaflet';
+import {
+  MapContainer as LeafletMapContainer,
+  TileLayer,
+  LayersControl,
+  FeatureGroup,
+  GeoJSON,
+  useMap
+} from 'react-leaflet';
 import MapEditControls from 'utils/MapEditControls';
 import { Feature } from 'geojson';
 import booleanEqual from '@turf/boolean-equal';
+import { LatLngBoundsExpression } from 'leaflet';
+
+export interface IMapBoundsProps {
+  bounds: LatLngBoundsExpression;
+}
+
+const MapBounds: React.FC<IMapBoundsProps> = (props) => {
+  const map = useMap();
+  const { bounds } = props;
+
+  map.fitBounds(bounds);
+
+  return null;
+};
```

Comment:
Whats this for?

Its a component, but it doesn't return any html. Could this just be a function instead?

Reply:
No I dont think it can be a function cause it uses a hook called useMap which is part of react leaflet and it expects it to be a child component of the leaflet map container